### DISCUSSION TITLE
HomeAssistant integration

### DISCUSSION THREAD

**Discussion by devdems** [tbnobody/OpenDTU#2358] [General]
I cannot believe that plug&play integration for OpenDTU -> HomeAssistant is not built. What are you using?

From what I could find, the options are MQTT and Rest API. Based on your experience, which one is more stable, meaning that I can get the data from the inverter to HA in <= 5s?

**Comment by stefan123t** [comment]
@devdems have you read the documentation on the MQTT Settings page ?
https://www.opendtu.solar/firmware/configuration/mqtt_settings/

Enable Home Assistant MQTT Auto Discovery 

Enables or disables the [Home Assistant MQTT Auto Discovery](https://www.home-assistant.io/integrations/mqtt/#mqtt-discovery) functionality.

**Comment by devdems** [comment]
@stefan123t and with MQTT, the system is able to process data on 5s intervals if the signal between the inverter and the board is ok?
